src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final int MAX_NUMBER = 101;
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
there is no need to make it static